modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestWithoutAccessToken.siddhi

```diff
@@ -0,0 +1,73 @@
+@App:name("PublishHttpOAuthRequest")
+@App:description("Send an OAuth events via HTTP transport without access token")
```

Comment:
Here also remove 'oauth events' part (we are sending HTTP events to an OAuth-protected endpoint).